command/vote_for_peace.rb

```diff
@@ -30,10 +30,11 @@ def execute(context:)
       game_data = context.game_data
 
       full_player_count = Player.all.count
+      allowed_vote_threshold = (full_player_count.to_f / 4.0).ceil
       alive_player_count = Player.where({ 'alive' => true }).count
 
-      if (alive_player_count.to_f/full_player_count.to_f) * 10 > 5
-        event.respond(content: "You cannot start a vote for peace with more than 50% of players alive", ephemeral: true)
+      if alive_player_count > allowed_vote_threshold
+        event.respond(content: "You cannot start a vote for peace with more than 25% (rounded up) of players alive", ephemeral: true)
```

Comment:
could you do like players.alive.count or something here? so people know how many people need to be alive? not sure if possible as not sure how you model players